api/client/service/update.go

```diff
@@ -30,8 +31,8 @@ func newUpdateCommand(dockerCli *client.DockerCli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.String("image", "", "Service image tag")
-	flags.StringSlice("command", []string{}, "Service command")
-	flags.StringSlice("arg", []string{}, "Service command args")
+	flags.String("command", "", "Service command")
+	flags.String("args", "", "Service command args")
```

Comment:
Also, the inconsistency with `service create` may be confusing. But again, I don't have a better suggestion. We could consider treating extra arguments that aren't associated with a flag as a replacement command/args, but this might be very unintuitive for other reasons (I pass an extra arg to `docker service update` by accident, and instead of getting an error, the command line for my service is overwritten...).